projects/python-client/src/datapane/client/api/user.py

```diff
@@ -113,6 +116,76 @@ def _run_script(script: str):
     runpy.run_path(str(script_path), run_name="__datapane__")
 
 
+def _run_template(template_path: str):
```

Comment:
use `pathlib.Path` instead of `str` where possible

Reply:
Updated to use `pathlib.Path` instead of `str. Also updated `def _run_template(template_path: str):` to def _run_template(template_path: Path):` and tested using `datapane hello-world`.